contracts/eosio.system/src/eosio.system.cpp

```diff
@@ -423,6 +424,66 @@ namespace eosiosystem {
       _global4.set( _gstate4, get_self() );
    }
 
+   void system_contract::setpayfactor( int64_t inflation_pay_factor, int64_t votepay_factor ) {
+      require_auth(get_self());
+      if ( inflation_pay_factor < pay_factor_precision ) {
+         check( false, "inflation_pay_factor must not be less than " + std::to_string(pay_factor_precision) );
+      }
+      if ( votepay_factor < pay_factor_precision ) {
+         check( false, "votepay_factor must not be less than " + std::to_string(pay_factor_precision) );
+      }
+      _gstate4.inflation_pay_factor = inflation_pay_factor;
+      _gstate4.votepay_factor       = votepay_factor;
+      _global4.set( _gstate4, get_self() );
+   }
+
+   void system_contract::setschedule( const time_point_sec start_time, double continuous_rate )
+   {
+      require_auth( get_self() );
+
+      check(continuous_rate >= 0, "continuous_rate can't be negative");
+      check(continuous_rate <= 1, "continuous_rate can't be over 100%");
```

Comment:
I believe the current system contract does allow for greater than +100% continuous rate

However, having this check is a good preventive measure in case the input is using the `annual_rate` type of numbers (ex: `annual_rate=300.0` would be 300% continuous rate instead of 3%)

Reply:
Yeah that's exactly where it came from, as the tests originally had higher numbers that caused bad outcomes and one slipped through, so this was a sanity check against them